packages/wordpress-external-dependencies-plugin/index.js

```diff
@@ -67,15 +71,23 @@ class WordPressExternalDependenciesPlugin {
 				for ( const c of entrypoint.chunks ) {
 					for ( const { userRequest } of c.modulesIterable ) {
 						if ( this.externalizedDeps.has( userRequest ) ) {
-							// Transform @wordpress deps:
-							//   @wordpress/i18n -> wp-i18n
-							//   @wordpress/escape-html -> wp-escape-html
-							// Pass other externalized deps as they are
-							entrypointExternalizedWpDeps.add(
-								userRequest.startsWith( WORDPRESS_NAMESPACE )
-									? 'wp-' + userRequest.substring( WORDPRESS_NAMESPACE.length )
-									: userRequest
-							);
+							let handle;
+
+							if ( userRequest.startsWith( WORDPRESS_NAMESPACE ) ) {
+								// Transform @wordpress deps:
+								//   @wordpress/i18n -> wp-i18n
+								//   @wordpress/escape-html -> wp-escape-html
+								handle = 'wp-' + userRequest.substring( WORDPRESS_NAMESPACE.length );
+							} else if ( 'tinymce/tinymce' === userRequest ) {
```

Comment:
Why is this block needed? Can we not rename so that the final `else` works fine?..

Reply:
See my comment above: https://github.com/Automattic/wp-calypso/pull/32162#discussion_r274237905